README.md

```diff
@@ -17,8 +17,9 @@ Installation Instructions for developing can be found
 
 ## Quick Start
 
-To quickly build and run all components you can use Docker. Make sure you have [Docker installed](https://www.docker.com/products/docker-desktop),
-then execute the following instructions inside the project directory:
+To quickly build and run all components you can use Docker. Make sure you have [Docker installed](https://www.docker.com/products/docker-desktop).
+Then you have to choose one of the profiles from below and set the required environment values.  
```

Comment:
Not a huge deal but a comment to simply execute set the env-variables in the same process or an example invocation wouldn't hurt either.